### PR TITLE
Added tabs system

### PR DESCRIPTION
**Pull request by inakijv**
Most important aspects:
- Added tabs system.

screen shot 2017-05-19 at 3 15 37 pm

### CODE REVIEW

**Comment by inakijv** [comment]
ping @SylvainGuittard

File: ui-guidelines/css/components-tabs.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+.ez-tabs ul li {
+    display: inline-block;
+    transition: all 0.2s ease-in;
+}
+
+.ez-tabs ul li:hover {
+    border-radius: 4px 4px 0 0;
+    background-color: #ccc;
+
```
Comment (SylvainGuittard) [inline ui-guidelines/css/components-tabs.css]: CS: empty line ;)

File: ui-guidelines/index.html [hunk 2]
```diff
@@ -982,6 +983,34 @@ <h4 class="u-move-down-xlarge">Modal with header and footer</h4>
...
 							</div>
 						</div>
+						<div id="components-tabs" class="modules">
+							<h3 class="module-header">Tabs</h3>
+							<h4 class="u-move-down-xlarge">Tabs system</h4>
+							<hr>
+							<div class="flex-wrapper">
+								<div class="container-tabs">
+									<div class="ez-tabs">
+					                    <ul class="tabs" role="tablist">
```
Comment (SylvainGuittard) [inline ui-guidelines/index.html]: Something is wrong with the indentation

File: ui-guidelines/index.html [hunk 3]
```diff
@@ -982,6 +983,34 @@ <h4 class="u-move-down-xlarge">Modal with header and footer</h4>
...
+					                        <li role="presentation" class="active">
```
Comment (SylvainGuittard) [inline ui-guidelines/index.html]: I am not sure about `role="presentation"`.
Should we keep it? Or change it?

**Comment by SylvainGuittard** [comment]
It seems that the code example part is missing. Am I right?

**Comment by inakijv** [comment]
Reviewed. Ping @SylvainGuittard

**Comment by inakijv** [comment]
Ping @SylvainGuittard just a reminder ;-)

**Comment by SylvainGuittard** [comment]
Thanks @inakijv

**Pull request merged**